TerrorConsole/Assets/_Main/Scripts/Tilemap/ShadowCaster2DCreator.cs

```diff
@@ -0,0 +1,90 @@
+using System.Linq;
+using UnityEngine;
+using System.Reflection;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+namespace TerrorConsole
+{
+    
+	#if UNITY_EDITOR
```

Comment:
This looks like an Editor class, you can remove this #if and leave the script below an "Editor" folder, we can check this today.